encoder_decoder_test.go

```diff
@@ -611,6 +612,24 @@ func TestEncoderLimits(t *testing.T) {
 			Input:       map[string]any{"bomb": selfPointer},
 			DecodeError: waferrors.ErrUnsupportedValue,
 		},
+		{
+			Name: "json-tagged",
+			Input: struct {
+				Ignored     string `json:"-"`
+				Transparent string `json:",omitempty"`
+				Renamed     string `json:"field"`
+			}{Ignored: "1", Transparent: "2", Renamed: "3"},
+			Output: map[string]any{"Transparent": "2", "field": "3"},
+		},
+		{
+			Name: "xml-tagged",
+			Input: struct {
+				xml.Name    `xml:"fancy"`
+				Transparent string `xml:",omitempty"`
+				Renamed     string `xml:"field,attr"`
+			}{Transparent: "2", Renamed: "3"},
+			Output: map[string]any{"Transparent": "2", "field": "3"},
```

Comment:
add ignored for completion?
```suggestion
			Input: struct {
				xml.Name    `xml:"fancy"`
				Ignored     string `xml:"-"`
				Transparent string `xml:",omitempty"`
				Renamed     string `xml:"field,attr"`
			}{Ignored: "2", Transparent: "3", Renamed: "4"},
			Output: map[string]any{"Transparent": "3", "field": "4"},
```